assets/js/modules/analytics/setup.js

```diff
@@ -674,6 +682,18 @@ class AnalyticsSetup extends Component {
 						</p>
 					</div>
 				}
+				{ onSettingsPage && useSnippet && ( ! ampEnabled || ( ampEnabled && ! ampClientIDOptIn ) ) && (
```

Comment:
This should only display if `ampMode !== 'primary'`. If someone is using secondary AMP, `ampEnabled` would be true, but there would still be a non-AMP version that this setting would affect.

Reply:
So the conditional here would be:

```js
onSettingsPage && useSnippet && ( ampMode !== 'primary' || ( ampMode === 'primary' && ! ampClientIDOptIn ) )
```

to clarify?

Reply:
Actually, I'm confused: wouldn't we always want to show this if the "Opt in AMP Client ID" setting is `false`? Eg if:

<img width="646" alt="Screenshot 2019-12-12 at 17 16 42" src="https://user-images.githubusercontent.com/90871/70733938-2c167c80-1d03-11ea-8a36-760a1d916edd.png">

is off we should show this.